packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -674,7 +728,23 @@ contract L1StandardBridge_FinalizeETHWithdrawal_Test is Bridge_Initializer {
     }
 }
 
-contract L1StandardBridge_FinalizeETHWithdrawal_TestFail is Bridge_Initializer { }
+contract L1StandardBridge_FinalizeETHWithdrawal_TestFail is Bridge_Initializer {
+    /// @dev Tests that finalizeETHWithdrawal reverts with custom gas token.
+    function test_finalizeETHWithdrawal_customGasToken_reverts() external {
+        vm.mockCall(
```

Comment:
This could be a good fuzz test

Reply:
I thought we'd just revert for all values with custom gas token? if so, I don't think it'd make sense to fuzz?